.github/workflows/pr.yml

```
@@ -202,6 +202,13 @@ jobs:
     uses: ./.github/workflows/e2e-atoms.yml
     secrets: inherit
 
+  api-v2-breaking-changes:
+    name: API v2 Breaking Changes
+    needs: [changes, build-api-v2]
+    if: ${{ needs.changes.outputs.has-files-requiring-all-checks == 'true' }}
+    uses: ./.github/workflows/api-v2-breaking-changes.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/api-v2-breaking-changes.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(expression)